Automattic/jetpack
# Newsletters: Make Subscriber Reach Sentence Future Tense for Scheduled Post

Fixes https://github.com/Automattic/jetpack/issues/31135


## Proposed changes:

Make newsletter subscribers reach sentence future-tense for scheduled posts

| Before | After |
|--------|--------|
| <img width="273" alt="Screenshot 2023-06-06 at 12 36 49 PM" src="https://github.com/Automattic/jetpack/assets/7076981/caaccaaa-8c2d-40c0-b494-e9734ae8cffa"> | <img width="278" alt="Screenshot 2023-06-06 at 12 55 40 PM" src="https://github.com/Automattic/jetpack/assets/7076981/22c3028b-cae4-4f36-8fa7-3419b933db83">


### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






1. Go to Posts > Add New
2. Write a post and save it as a draft.
3. Set a publishing date for the future
4. Schedule your post
5. Notice the Prepublish panel displays the subscribers reach in the future tense
